actor/src/main/resources/reference.conf

```diff
@@ -323,8 +323,8 @@ pekko {
           # Probability of doing an exploration v.s. optimization.
           chance-of-exploration = 0.4
 
-          # When downsizing after a long streak of under-utilization, the resizer
-          # will downsize the pool to the highest utilization multiplied by a
+          # When downsizing after a long streak of underutilization, the resizer
+          # will downsize the pool to the highest utiliziation multiplied by a
```

Comment:
the next line also has an `a`, remove one

Reply:
Thanks, this is a new typo found.